README.md

```diff
@@ -0,0 +1,48 @@
+# 🔥 LIT Scheduled Tasks 🔧
+
+This repo provides scaffolding for running internal scheduled tasks on a CRON-like schedule.
+
+## ❌ Installation
+
+Packages in this repository are currently not published to NPM; it is expected to be deployed as a worker to Heroku.
+
+## 🎮 Usage
+
+#### To prepare for either production usage or local development, you must install dependencies and build the packages.
+
+```zsh
```

Comment:
Just curious, why `zsh` and not `bash`?

Reply:
Webstorm provide super helpful little 'run' buttons when viewing this stuff in the IDE, and it uses the shell provided to run it... and zsh is the most popular these days so I'm just betting on commonality